libraries/lib-datahandler/src/test/java/org/eclipse/sw360/datahandler/couchdb/AttachmentStreamConnectorTest.java

```diff
@@ -23,6 +26,7 @@
 import java.util.concurrent.TimeUnit;
 
 import static org.eclipse.sw360.datahandler.common.Duration.durationOf;
+import static org.hamcrest.CoreMatchers.any;
```

Comment:
This import is unused and the references to `Mockito.any` in lines 70 and 73 are unnecessarily qualified.